tests/functional/test_chain.py

```diff
@@ -95,8 +95,8 @@ def test_account_history(sender, receiver, chain):
     assert len(transactions_from_cache) == length_at_start + 1
 
     txn = transactions_from_cache[-1]
-    assert txn.sender == receipt.sender == sender
-    assert txn.receiver == receipt.receiver == receiver
+    assert txn.raw_transaction.sender == receipt.raw_transaction.sender == sender
```

Comment:
Wonder if you could make properties in `ReceiptAPI` that re-expose properties in `TransactionAPI` via `__getattr__` for exactly these use cases

Reply:
we can make `transaction` hidden by defining the attribute as `_transaction`. Then we can define a method for each value in `_transaction` to get unpacked and define each attribute that is defined in `transaction` into `receipt`

Reply:
yes, I think let's do this. add a `__getattr__` method to `ReceiptAPI` that looks up fields via `getattr` in `.transaction`

Reply:
[4f12806](https://github.com/ApeWorX/ape/pull/1015/commits/4f1280606c1c84d2372599fd4252f3dd07c5cb28)